airbyte-cdk/python/unit_tests/sources/declarative/requesters/request_options/test_interpolated_request_options_provider.py

```diff
@@ -84,9 +84,3 @@ def test_error_on_create_for_both_request_json_and_data():
     request_data = "interpolate_me=5&invalid={{ config['option'] }}"
     with pytest.raises(ValueError):
         InterpolatedRequestOptionsProvider(config=config, request_body_json=request_json, request_body_data=request_data)
-
-
-def test_interpolated_request_kwargs_is_empty():
```

Comment:
this test was moved to https://github.com/airbytehq/airbyte/pull/14973/files#diff-eb7a9110211a5b84c7039ade89d0b0385d480ba9192a41f7a280f7809b6fecefR61